packages/dp-auctions-lib/src/index.js

```diff
@@ -9,7 +9,7 @@ const pTap = require('p-tap').default
 const { findToken } = require('./token-list')
 const dpaAbi = require('./abi.json')
 
-const DPA_ADDRESS = '0x164D41ceB60489D2e054394Fc05ED1894Db3898a' // Chain ID 1
+const DEFAULT_DPA_ADDRESS = '0x164D41ceB60489D2e054394Fc05ED1894Db3898a' // Chain ID 1
```

Comment:
I believe this one should also be moved into env variables (or a Json array with addresses per chain). This should be for all addresses that are currently hardcoded for Mainnet, so later on it's easy to deploy a new env for either Hemi or Ethereum

Reply:
if this address is on Hemi (as this PR goes into this fork and not upstream), then the comment only needs to be updated

```suggestion
const DEFAULT_DPA_ADDRESS = '0x164D41ceB60489D2e054394Fc05ED1894Db3898a' // Hemi
```

or 
```suggestion
const DEFAULT_DPA_ADDRESS = '0x164D41ceB60489D2e054394Fc05ED1894Db3898a' // Chain ID 743111
```

(Or even the Hemi mainnet chainId, which I don't remember and can't find now)

or even removing it altogether!

```suggestion
const DEFAULT_DPA_ADDRESS = '0x164D41ceB60489D2e054394Fc05ED1894Db3898a'
```

Reply:
JSON file > env vars > hardcoded addresses!

Reply:
Removed env vars and left it as a JSON with the chainIds as keys. I left mainnet addresses empty for now as I don't know them. Here: https://github.com/hemilabs/pure.finance/pull/2/commits/50613c94ab27cb3cc111f820f6ba587e14cf0d97#diff-2660642bc5060d28c91a9c692dd7f76f88758edb5f55f4e7df813f35f510e1c2